### PR TITLE
Readme Update based on recent Documentation Change

### PR DESCRIPTION
**Pull request by tangy5**
Signed-off-by: tangy5 <yucheng.tang@vanderbilt.edu>

Updated the MONAI Label Github Readme.md according to our latest documentation changes. 

Added anchor link for the readme.
Made the style consistently with MONAI Label documentation page. 
Added description of Bundle Apps, Endoscopy 
Added visualization tools guide.

### CODE REVIEW

**Review by Nic-Ma — APPROVED** [review]
Hi @yuchen-xu ,

Looks good to me, could you please help add links of your updated doc pages to this README?
Then users can easily jump there.

Thanks in advance.

**Comment by zmsunnyday** [comment]
Hi @tangy5 , 
Very clear readme, and installation guidance. Thanks for your excellent work! I guess these two should be in bold.
image

Best regards,
Meng

**Comment by tangy5** [comment]
> Hi @tangy5 , Very clear readme, and installation guidance. Thanks for your excellent work! I guess these two should be in bold. image
> 
> Best regards, Meng

Thank you! Made them bold and updated!

**Comment by tangy5** [comment]
> Hi @yuchen-xu ,
> 
> Looks good to me, could you please help add links of your updated doc pages to this README? Then users can easily jump there.
> 
> Thanks in advance.

Added the link in the beginning MONAI Label introduction.

Thanks

File: README.md [hunk 1]
```diff
@@ -123,15 +160,40 @@ Following are the optional dependencies which can help you to accelerate some GP
 These dependencies are by-default available if you are using `projectmonai/monailabel` docker.
 - [CUCIM](https://pypi.org/project/cucim/)
 - [CUPY](https://docs.cupy.dev/en/stable/install.html#installing-cupy)
-- [CUDA Toolkit](https://developer.nvidia.com/cuda-downloads)
```
[the remark this answers is not in the document]
Comment (tangy5) [reply README.md:126]: Fixed, thanks!

[the remark this answers is not in the document]
Comment (SachidanandAlle) [reply README.md:126]: i don't see them yet back.. in the final PR changes...

**Review by SachidanandAlle — APPROVED** [review]

**Pull request merged**